pkg/asset/internal/templates.go

```diff
@@ -129,7 +129,7 @@ spec:
       hostNetwork: true
       containers:
       - name: checkpoint-installer
-        image: quay.io/coreos/pod-checkpointer:91d5a311eee40d579a8f0549c10eeed57979d6c4
+        image: kenan435/pod-checkpointer:v1.0.0
```

Comment:
Let's continue using the official image, the tag will need to be updated in a follow-up PR, and another official image built.